app/javascript/packs/tpi.js

```diff
@@ -0,0 +1,14 @@
+import Turbolinks from "turbolinks";
+import Chartkick from "chartkick";
+import HighCharts from "highcharts";
+import Rails from "rails-ujs";
```

Comment:
We should review this later since this stuff adds jQuery to the bundle and, with React already in, it's useless and waaay to much to have in the bundle.

Reply:
Yeah, I feel guilty of adding jquery to the frontend bundle, but I guess it's still hard to avoid it as even many react libraries are just wrappers on jquery plugins. I added as I wanted `html` function to replaceHtml and eval scripts, maybe I could do it differently. Simple changing `innerHtml` of element is not enough.

Reply:
`even many react libraries are just wrappers on jquery plugins` only the bad, unmaintained ones.
But we can deal with it later on.